.github/workflows/qat.yml

```diff
@@ -14,12 +26,59 @@ permissions:
 jobs:
   playwright:
     name: Playwright
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
The current latest ubuntu (22.04) doesn't work with `npx playwright install --with-deps` but 20.04
Related link - https://github.com/microsoft/playwright/issues/13738